Automattic/wp-calypso
# Stats: Deprecate the feature flag stats/new-main-chart

#### Proposed Changes

* Refactor a few modernized components to coexist with legacy styles.
* Deprecate the legacy HTML structure in pages and components.

#### Testing Instructions



> Note: The instructions to show the `Store` and `Ads` pages can be found at https://github.com/Automattic/wp-calypso/pull/69661 and https://github.com/Automattic/wp-calypso/pull/69743.

* Spin up this change with the Calypso Live link.
* Navigate to the Stats page `Traffic`, `Insights`, `Store`, and `Ads`.
* Ensure the styles are the same as the previous ones.

#### Pre-merge Checklist





Related to #70209
